app/armour.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import annotations
+
+
+class Armour:
+    def __init__(self, name: str, protection: int) -> None:
+        self.name = name
+        self.protection = protection
+
+    def __iadd__(self, other: Armour) -> int:
```

Comment:
So this method adds stats from one armour to another but wouldn't it make more sense to add those stats to a knight?